### PR TITLE
Add python quickstart

### PR DESCRIPTION
**Pull request by dsmilkov**
https://lilacml.com/getting_started/quickstart_python.html

### CODE REVIEW

File: docs/getting_started/quickstart_python.md [hunk 1]
```diff
@@ -4,8 +4,130 @@
...
+
+The dataset now has extra fields for each of the computed signal:
+
+```py
+print(dataset.select_rows(limit=5).df())
+```
+
+```sh
+             id                                      system_prompt                                           question  ...                         __rowid__                                       question.pii                                       response.pii
+0    flan.77076  You are an AI assistant. User will you give yo...  How does the sentence end?\n\n(CNN) -- When sh...  ...  91885e6e7ff7490687fc3cd23e43eb3f  {'emails': [], 'ip_addresses': [], 'secrets': []}  {'emails': [], 'ip_addresses': [], 'secrets': []}
```
Comment (nsthorat) [inline docs/getting_started/quickstart_python.md:79]: I feel like the statement would be stronger / more informative if there were actually some PII here, could we choose a filter where we see something?

File: docs/getting_started/quickstart_python.md [hunk 2]
```diff
@@ -4,8 +4,130 @@
...
+
+Let's also run the profanity concept on the `response` field to see if the LLM produced any profane
+content. To see the results, we need to _index_ the `response` field using a text embedding. We only
+need to index once. For a fast on-device embedding, we recommend the
+[GTE-Small embedding](https://huggingface.co/thenlper/gte-small).
+
+Before we can index with GTE-small, we need to install the optional dependencies:
+
+```sh
+pip install sentence-transformers
```
Comment (nsthorat) [inline docs/getting_started/quickstart_python.md:98]: how do we communicate this generally? is it possible for us to do this as an extra specifically for gte?

File: docs/getting_started/quickstart_python.md [hunk 3]
```diff
@@ -4,8 +4,130 @@
...
+PII detection. To install the optional pii, run:
+
+```sh
+pip install lilacai[pii]
+```
+
+Let's run the PII detection signal on both the `question` and the `response` field.
+
+```python
+dataset = ll.get_dataset('local', 'open-orca-100k')
```
Comment (nsthorat) [inline docs/getting_started/quickstart_python.md:57]: nit: add an import here? it will bridge the lilac => ll

**Review by nsthorat — APPROVED** [review]

**Comment by dsmilkov** [comment]
did all the comments, new version deployed at https://lilacml.com/getting_started/quickstart_python.html

**Pull request merged**